### DISCUSSION TITLE
🎉 Loco version 0.8.0 was released!

### DISCUSSION THREAD

**Discussion by jondot** [loco-rs/loco#726] [Announcements]
- Added: `loco-cli` (loco new) now receives options from CLI and/or interactively asks for configuration options such as which asset pipeline, background worker type, or database provider to use.
![image](https://github.com/user-attachments/assets/8064aa26-4cce-4db9-932b-dc16b056ca33)

- Fix: custom queue names now merge with default queues.
- Added `remote_ip` middleware for resolving client remote IP when under a proxy or loadbalancer, similar to the Rails remote_ip middleware.
- Added `secure_headers` middleware for setting secure headers by default, similar to how https://github.com/github/secure_headers works. This is now ON by default to promote security-by-default.
- Added: `money`, `blob` types to entities generator.
- Doc fixes
- Dependencies upgrades

Thanks to:  @mellkior @fan-tastic-z @katayama8000 @ealmloff @lerina @not4rt for your contributions 👍

**Comment by bhagdave** [comment]
This breaks the chat-room example channels with `MessageHandler<LocalAdapter, _>` is not satisfied`  Issue created - https://github.com/loco-rs/loco/issues/743
